Doughnut/Library/Episode.swift

```diff
@@ -184,6 +184,11 @@ class Episode: Record {
     pubDate = feedItem.pubDate
     link = feedItem.link
 
+    let duration = feedItem.iTunes?.iTunesDuration
+    if (duration != nil) {
```

Comment:
It's generally recommended to omit the parentheses around if conditionals for Swift code.